<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inventory_ui</class>
 <widget class="QWidget" name="inventory_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="grassRB">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Grass</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dirtRB">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Dirt</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="stoneRB">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Stone</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="waterRB">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Water</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="snowRB">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Snow</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="bedrockRB">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bedrock</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="lavaRB">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Lava</string>
   </property>
  </widget>
  <widget class="QLabel" name="grassLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="dirtLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="stoneLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="waterLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="lavaLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="snowLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="bedrockLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Inventory! Choose your block</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
